beyond-essence/inventory/toggle-switches updated.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: beyond-essence
+header_image: private/main.jpg
+title: Toggle switches
+background_image: index.jpg
+---
+{% image_custom image="/assets/img/pages/essence/switch-button-on-off.png" width="25" align="right" %}
+A toggle switch works like a physical switch, that can be used e.g. to turn the light on and off.
+In the cockpit we display toggle switches to enable merchants to turn a feature on or off, and to enhance cards with additional features.
+
+The UI element "checkbox" is used quite similarly.
+Check the section "Choosing between toggle switch and checkbox" below and have a look at the inventory entry [Checkboxes](/beyond-essence/inventory/checkboxes/) to find the right element for your use case.
+
+## Use cases
+
+In the cockpit, toggle switches have two main use cases:
+
+### Activate significant features
+
+Some toggle switches in the cockpit have a significant impact on the functionality of the cockpit and the merchant's workflow.
+When their default state is changed, typically more than one part of the cockpit is impacted.
+
+**Example: Website settings**
+
+* **Website is live**: When the merchant activates this toggle, the website goes live and will be visible to customers.
+Customers will be able to view and interact with the merchant's website. This is an important step for the merchant, as they need to make sure they're all set before activating the toggle.
+* **Allow visitors to make purchases on your website**: When the merchant activates this toggle, a cart will be displayed at the top of the website and each product will have an "Add to cart" button, allowing visitors to order the merchant's products.
+The merchant needs to make sure they are ready to sell (and not only showcase) their products on their website before activating the toggle.
+
+💡 _Note: This is a temporary solution and might be handled differently in the future._
+
+### Enhance cards with extensions
+
+The cockpit is customizable.
+In order to allow merchants to adapt the cockpit to their needs, they can turn features on or off.
+This is possible, for example, on single cards.
+
+**Example: Enable pricing for variations**
+
+For the pricing of variation products, the merchant has two options:
+
+* If the toggle is _disabled_ (default): The same price will be applied to all variations 
+* If the toggle is _enabled_: A table opens on the same card and allows the merchant to set an individual price for each variation.
+
+Both options don't have an impact on the functionality of the other cards, but enhance or reduce the functionality of the Pricing card only.
+
+## Structure
+
+A toggle switch looks like a classical on/off switch.
+The two different states give a clear visual feedback, so that the merchant is always aware if the toggle switch is turned on or off.
+
+* A checkmark on a green background, toggled to the right, represents an active feature.
+* A cross on a beige background, toggled to the left, represents a deactivated feature.
```

Comment:
The Structure section is unfortunately not accurate for the manage variations delete mode toggle :( :( :(